helper.py

```diff
@@ -249,6 +249,18 @@ def extract_floating_from_string(value):
     except ValueError as _ve:
         logging.exception(constant.VALUE_ERROR_EXCEPTION_RAISED, _ve)
 
+    return _result
+def str_to_float(value):
+    """Convert string to Float, if possible"""
+    _result = 0
+    try:
+        if (value is not None and value != ''):
+            _exp_result = re.sub('[^\d\.]', '', value)
+            _result = (float(_exp_result))
+
+    except ValueError as _ve:
+        logging.exception(constant.VALUE_ERROR_EXCEPTION_RAISED, _ve)
+
```

Comment:
would it be better if `None` is returned than `0`

Reply:
Yeah, fair point. I've update that function to default to `None`. Additionally, the `create_soil_volumes` section that's calling this has been updated to include a check for None and a `continue` so we don't get any unwanted math errors